straxen/analyses/quick_checks.py

```diff
@@ -196,15 +196,17 @@ def plot_energy_spectrum(
         if exposure_kg_sec is not None:
             unit = 'kg_day_kev'
         else:
-            unit = 'events'       
-    
+            unit = 'events'
```

Comment:
Not number of events or short '#events' ?

Reply:
as this is only internal I don't think its much of an issue, I actually did not even know we had this function to be honest